src/main/pages/che-7/extensions/assembly_eclipse-che4z.adoc

```diff
@@ -21,7 +21,7 @@ Developers can now find the code they need to work on in Explorer for Endevor, G
 
 Eclipse Che4z is an open source project. Many of the components of the Che4z stack are also available on the VS Code marketplace as part of the https://marketplace.visualstudio.com/items?itemName=broadcomMFD.code4z-extension-pack[Code4z] package.
 
-== Use Cases
+== Use cases
```

Comment:
I'm pretty sure I fixed this one... Don't forget to rebase from time to time so we don't do double work. :D